diff --git a/build.gradle b/build.gradle
index 1cbab29..dbc65b8 100644
--- a/build.gradle
+++ b/build.gradle
@@ -23,6 +23,10 @@ buildscript {
 def slf4jlog4j='org.slf4j:slf4j-log4j12:1.7.6'
 
 allprojects {
+  apply plugin: 'java'
+  sourceCompatibility = 1.6
+  targetCompatibility = 1.6
+
   apply plugin: 'idea'
   repositories {
     mavenCentral()
@@ -49,6 +53,14 @@ subprojects {
 
   licenseTest.onlyIf { isVerificationRequired(project) }
 
+  configurations {
+    deployerJars
+  }
+
+  dependencies {
+    deployerJars "org.springframework.build:aws-maven:5.0.0.RELEASE"
+  }
+
   uploadArchives {
     repositories {
       signing {
@@ -58,9 +70,9 @@ subprojects {
               // To test locally, replace mavenUrl in ~/.gradle/gradle.properties to file://localhost/tmp/myRepo/
               mavenDeployer {
                   beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-                  repository(url: "${mavenUrl}") {
-                      authentication(userName: "${mavenUsername}", password: "${mavenPassword}")
-                  }
+                  configuration = configurations.deployerJars
+                  repository(url: "REPOSITORY")
+                  snapshotRepository(url: "SNAPSHOT_REPOSITORY")
                   afterEvaluate {
                       pom.artifactId = "${archivesBaseName}"
                       pom.project {
